code/DocumentationManifest.php

```diff
@@ -609,6 +609,16 @@ public function normalizeUrl($url)
     }
 
     /**
+     * @param string
+     *
+     * @return string
+     */
+    public function normalizePath($path)
+    {
+        return str_replace('\\', '/', $path);
```

Comment:
I'm concerned that this may be fragile to different dir separators.

Can we use the `DIRECTORY_SEPARATOR` on this instead of hard coding `'\\'`? We could check if the dir seperator is not `/` and only then do the replacement?